File: vstemplates/templates-2008/VBSQlClrAggregate/SqlClrAggregate.vb
Imports System
Imports System.Data.Sql
Imports System.Data.SqlTypes
Imports System.Runtime.InteropServices
Imports Microsoft.SqlServer.Server
Imports DM.Build.Yukon.Attributes


<Serializable()> _
<StructLayout(LayoutKind.Sequential)> _
<SqlUserDefinedAggregate(Format.Native)> _
Public Class $safeitemrootname$

  Public Sub Init()

  End Sub


  Public Sub Accumulate(ByVal Value As SqlString)

  End Sub


  Public Sub Merge(ByVal Group As $safeitemrootname$)

  End Sub

  Public Function Terminate() As SqlString

    Return ""
  End Function
End Class

File: vstemplates/templates-2008/CsSqlClrAggregate/SqlClrAggregate.cs
using System;
using System.Data.Sql;
using System.Data.SqlTypes;
using System.Runtime.InteropServices;
using Microsoft.SqlServer.Server;
using DM.Build.Yukon.Attributes;


[Serializable]
[StructLayout(LayoutKind.Sequential)]
[SqlUserDefinedAggregate(Format.Native)]
public class $safeitemrootname$ {
  public void Init() {
    
  }

  public void Accumulate(SqlString Value) {
    
  }

  public void Merge ($safeitemrootname$ Group) {
    
  }

  public SqlString Terminate () {
    
    return "";
  }
}
